playbooks/compute/nto/roles/configurecluster/tasks/ppc.yml

```diff
@@ -0,0 +1,62 @@
+---
+- name: Create performance Profile
+  ansible.builtin.include_tasks: create_directory.yml
+  vars:
+    dir_path: "{{ must_gather_dir }}"
+
+- name: Collect Must Gather
+  environment:
+    KUBECONFIG: "{{ kubeconfig }}"
+  ansible.builtin.shell: |
+    oc adm must-gather --dest-dir={{ must_gather_dir }}
+  register: must_gather
+  changed_when: must_gather.rc != 0
```

Comment:
I'm not sure that's what you want here `This means the task will only be marked as "changed" when the command fails `

I think changed_when: true is better

Reply:
or == 0 is also good. depends